src/components/EditorsColumn.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import isEmpty from 'lodash/isEmpty';
+import includes from 'lodash/includes';
+import partial from 'lodash/partial';
+
+import EditorContainer from './EditorContainer';
+import Editor from './Editor';
+
+class EditorsColumn extends React.Component {
+  _allErrorsFor(language) {
+    if (language === 'javascript') {
+      return this.props.errors.javascript.items.
+      concat(this.props.runtimeErrors);
+    }
+
+    return this.props.errors[language].items;
+  }
+
+  render() {
```

Comment:
Gotcha. Yeah, my inclination would be to go with an `allErrorsFor` function, which has the appeal of module encapsulation as well.